tests/testthat/test-scatterplot.R

```diff
@@ -78,7 +79,7 @@ test_that("scatterplot module works as expected in the test app", {
   expect_identical(res, "GeneID:8086")
 
   # Remove sample filter
-  app$click("teal-teal_modules-scatterplot-filter_panel-filters-MAE-subjects-MAE_SEX-remove")
+  app$click(ns_fp("MAE-subjects-MAE_SEX-remove"))
   app$wait_for_idle()
 
   res <- app$get_value(input = ns("x_spec-genes"))
```

Comment:
Not directly related to this change, but in line 91

`app$set_inputs(!!ns("experiment-name") := "hd2")` is redundant and has no effect as it is already set before in line 42

This appears on the error

<img width="1153" height="206" alt="image" src="https://github.com/user-attachments/assets/a3b20b41-4510-430a-88c3-a18d03dab3e3" />

I recommend to remove it or create a way to skip if already "hd2"